app/main.py

```diff
@@ -1,3 +1,19 @@
+from math import ceil
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+        self.days_to_weeks(13)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return cls(name=course_dict["name"],
+                   description=course_dict["description"],
+                   weeks=cls.days_to_weeks(course_dict["days"]))
```

Comment:
It would be better to use `OnlineCourse.days_to_weeks` here. You will learn about this case in the future videos on OOP